source/setup/js/UnlockArchive.jsx

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 import { hashHistory } from "react-router";
+import { noop } from "lodash";
```

Comment:
There's no need to import lodash here. `NOOP` (playfully renamed to `NOPE` here) is classic nomenclature for "no operation", and is commonly and empty function. We don't need a library to create an empty function 😅.

Reply:
I just saw `lodash` in `node_modules` and decided to use it :smile:.
And yes, I agree, there is no need to do that.

btw, travis-ci is still failing on this step - `webpack --progress` + I don't have errors locally.